spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/JobExecutionController.java

```diff
@@ -93,6 +93,25 @@ public JobExecutionController(TaskJobService taskJobService) {
 		return assembler.toResource(page, jobAssembler);
 	}
 
+	/**
+	 * Return a page-able list of {@link JobExecutionResource} defined jobs that
+	 * do not contain step execution detail.
+	 *
+	 * @param pageable page-able collection of {@code TaskJobExecution}s.
+	 * @param assembler for the {@link TaskJobExecution}s
+	 * @return a list of Task/Job executions(job executions do not contain step executions.
+	 * @throws NoSuchJobExecutionException in the event that a job execution id specified
+	 * is not present when looking up stepExecutions for the result.
+	 */
+	@RequestMapping(value = "/jobexecutioninfoonly", method = RequestMethod.GET, produces = "application/json")
```

Comment:
Instead of a separate REST endpoint, can we have a new request parameter with the name similar to `excludeStepExecutions`  to the existing `jobs/executions` endpoint?

Moreover the name `/jobexecutioninfoonly` highlights the word `foo` than `info only` ;)

Reply:
done